pkg/descheduler/strategies/lownodeutilization_test.go

```diff
@@ -54,14 +54,12 @@ func TestLowNodeUtilization(t *testing.T) {
 		thresholds, targetThresholds api.ResourceThresholds
 		nodes                        map[string]*v1.Node
 		pods                         map[string]*v1.PodList
-		// TODO: divide expectedPodsEvicted into two params like other tests
-		// expectedPodsEvicted should be the result num of pods that this testCase expected but now it represents both
-		// MaxNoOfPodsToEvictPerNode and the test's expected result
-		expectedPodsEvicted int
-		evictedPods         []string
+		maxPodsToEvictPerNode        int
+		expectedPodsEvicted          int
+		evictedPods                  []string
 	}{
 		{
-			name: "no evictable pods",
+			name: "no evictable pods -- Max Evict Pods Num 0",
```

Comment:
Also, no need to mention `.. Max Evict Pods Num ...`. `0` means the test for maximum number of pods to evict per node is not checked.